cli/assets/templates/grain/app/main.gr

```diff
@@ -0,0 +1,49 @@
+module HelloWorld
+
+from "wasm4" include Wasm4 as W4
+from "uint8" include Uint8
+use Uint8.{ (+), (-), (&), (>), (==) }
+
+let smiley = b"\xc3\x81\x24\x24\x00\x24\x99\xc3"
```

Comment:
I tested a variation of this example using png2src output instead of the manual things here, and it worked for me.

The original example here also worked for me. I used a manually built Grain 0.6.3.